skills/lookup.js

```diff
@@ -0,0 +1,53 @@
+const ExpressionList = require('../submodules/Regex/ExpressionList');
+const ping = require('ping');
+
+
+module.exports = controller => {
+    controller.hears(
+        [ExpressionList.lookup], 
+        'direct_message', 
+        async (bot, message) => {
+            bot.createConversation(message, async(err, convo) => {
+                if (err) return false;
+                var domain = message.text.split(' ')[1];
```

Comment:
Technically this works pretty well, but if any flakey syntax is tried by accident (like extra spaces) this is likely to fail, speaking from experience in the other skills.

Instead, I would change the regex to something like this:
`lookup: /.*(?:lookup)\s{0,3}(.*)/i`

This means:
- anything then the exact work "lookup" (first match)
- followed by 0-3 spaces
- followed by anything (second match)

The beauty of this is anything after the word lookup with up to 3 spaces is treated as the second match. In this engine, you can pull out regex matches (anything between the capturing groups wrapped in `(...)` symbols in the regex) by looking at `message.match`, which contains an array of matches:
<img width="332" alt="Screenshot 2020-01-13 at 11 39 10" src="https://user-images.githubusercontent.com/14090872/72253421-6e302800-35f9-11ea-8ac3-fa9d6e34f91d.png">

This also means you can replace this:
```
var domain = message.text.split(' ')[1];
```
with
```
var domain = message.match[1];
```

Reply:
Second point - if you plan on changing a variable after you've created it, use `let`. If you only plan to store something and never plan on changing it, use `const`. Here you're changing domain further down, so would recommend using `let`